test/e2e/node/gpu.go

```diff
@@ -178,7 +179,20 @@ func testNvidiaCLIPod() *v1.Pod {
 			Containers: []v1.Container{
 				{
 					Name:  "nvidia-smi",
-					Image: "nvidia/cuda:12.3.2-runtime-ubuntu22.04",
+					Image: "nvidia/cuda:12.5.0-devel-ubuntu22.04",
+					Command: []string{
+						"bash",
+						"-c",
+						`
+nvidia-smi
+apt-get update -y && \
+	DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated cuda-demo-suite-12-5
+/usr/local/cuda/extras/demo_suite/deviceQuery
```

Comment:
Given you have added these additional stuff, wondering if we want to check these demo stuff in logs ?

`- gomega.Expect(log).To(gomega.ContainSubstring("CUDA Capability")) ` etc  for these additional stuff ?

Reply:
(as you iterate on this in future versions)

Reply:
sounds good!

Reply:
https://github.com/NVIDIA/cuda-samples/blob/master/Samples/1_Utilities/deviceQuery/deviceQuery.cpp#L109